src/block/input.css

```diff
@@ -0,0 +1,31 @@
+.search-form__input {
+  height: 100%;
+  width: 100%;
+  padding: 0;
+
+  font-size: 16px;
+
+  border: none;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+}
+
+.search-form__input--small {
+  font-size: 14px;
+}
+
+.search-form__input:focus {
+  outline: none;
+  font-weight: 900;
```

Comment:
The use of `font-weight: 900;` on focus might lead to a text reflow which can be distracting for users. Consider maintaining consistent font weight or using other visual cues for focus state that do not affect the layout or text flow.